### DISCUSSION TITLE
Looking for a recommendation to get started using this

### DISCUSSION THREAD

**Discussion by shmup** [kelleyma49/PSFzf#184] [Q&A] [answered]
I'd like to incorporate this into my _very new_ Windows workflow. Linux background, love my fzf setup there, want to get some or this projects niceties into my Powershell.

So, where does one start? Must I build this solution? I've tried that, to see if a binary was produced.

Wasn't sure if I'm just to include these scripts in my $PATH or whatever. 

If this information is right in front of my eyes in the README.md, please just point me there hehe. Or, maybe it's in /docs, I haven't looked all that well

Cheers,

Not Strong At Windows, Yet

**Comment by pluma9** [comment, marked as answer]
I hope this answer is not too late. The powershellgallery.com URL in the README implies that you can install the module by just running: `Install-Module PSFzf`. Once done, the commands will be available.
